image-layout.md

```diff
@@ -0,0 +1,55 @@
+## Open Container Initiative Image Layout Specification
+
+The OCI Image Layout is a slash separated layout of OCI content-addressable blobs and human significant [location-addressable](https://en.wikipedia.org/wiki/Content-addressable_storage#Content-addressed_vs._location-addressed) references (refs).
+This layout MAY be used in a variety of different transport mechanisms: archive formats (e.g. tar, zip), shared filesystem environments (e.g. nfs), or networked file fetching (e.g. http, ftp, rsync).
+Given an image layout a tool can convert a given ref into a runnable OCI Image Format by finding an appopriate manifest from the manifest list, unpacking the filesystem serializations in the correct order, and then converting the image configuration into an OCI Runtime config.json.
+
+The image layout has two top level directories:
+
+- "blobs" contains content-addressable blobs. A blob has no schema and should be considered opaque. It is addressed by its hashed contents from byte 0 to the end of the file.
```

Comment:
Can the last sentence be “The full blob is hashed as it is stored.”?  I understand you're trying to rule out headers, compression, etc., but “from byte 0 to the end of the file” is a bunch of words in a somewhat tangential direction.  Maybe post an example (`sha256sum blobs/sha256-aff…` → `aff…`)?